lib/src/command/global_activate.dart

```diff
@@ -90,7 +88,7 @@ class GlobalActivateCommand extends PubCommand {
 
     Iterable<String> args = argResults.rest;
 
-    dynamic readArg([String error]) {
+    dynamic readArg([String error = '']) {
```

Comment:
We should be able to return a String here (perhaps not this PR?)
```suggestion
    String readArg([String error = '']) {
```

Reply:
done